library/os_container_infra.py

```diff
@@ -105,8 +105,8 @@ def connect(self, cloud):
         self.client = Client('1', endpoint_override=False, session=self.cloud.session)
 
     def apply(self):
-        tries = 0
-        fail_after = 100
+        timeout = 3600
```

Comment:
why are we using absolute time rather than retries?

Reply:
It's more accurate, and easier to reason about. We don't really care how many times you asked about the cluster status, we care how long it's taken. Less important is the variability introduced by the duration of API calls, which is not included in the timing with retries.